crates/block-producer/src/withdrawal_unlocker.rs

```diff
@@ -256,7 +264,8 @@ impl BuildUnlockWithdrawalToOwner for DefaultUnlocker {
     }
 
     async fn query_rollup_cell(&self) -> Result<Option<CellInfo>> {
-        self.rpc_client.query_rollup_cell().await
+        let local_cells_manager = self.local_cells_manager.lock().await;
+        query_rollup_cell(&local_cells_manager, &self.rpc_client).await
```

Comment:
Because PSC's l1transactions may or may not be sent to CKB, it seems that the withdrawal unlocker may reference a not-sent out-point?

Reply:
Unlocker will retry later